src/libcollections/vec.rs

```diff
@@ -456,8 +458,9 @@ impl<T> Vec<T> {
     }
 
     /// Reserves the minimum capacity for exactly `additional` more elements to
-    /// be inserted in the given `Vec<T>`. Does nothing if the capacity is already
-    /// sufficient.
+    /// be inserted in the given `Vec<T>`. After calling `reserve_exact`,
+    /// capacity will be greater than or equal to `self.len() + additional`.
```

Comment:
@sgrif Can you change your comment here (and ideally also the example) to say "equal"/`==` instead of "greater than or equal"/`>=`? Just don't want to mislead people about how this API works.